app/presenters/track.rb

```diff
@@ -0,0 +1,39 @@
+module ExercismWeb
+  module Presenters
+    class Track
+
+      def initialize(track_id)
+        @track_id = track_id
+        @track = Trackler.tracks[track_id]
+      end
+
+      def method_missing(method, *args)
+        @track.send(method, *args)
```

Comment:
Hmm, I was indifferent here, so I went ahead and made the change as part of [this commit](https://github.com/exercism/exercism.io/pull/3454/commits/ba046e7aa4890e201aa998002b9cc1e6e481e73c).

Either way, we'll raise an exception if neither the class or `@track` responds to the method, right? What's the benefit of raising that exception here vs. in `Trackler`?